FRAuth/FRAuth/FRAuth.swift

```diff
@@ -228,6 +228,18 @@ public final class FRAuth: NSObject {
     
     
     // - MARK: Private
+    
+    /// Initiates Authentication Tree with `suspendedId`
+    /// - Parameters:
+    ///   - suspendedId: suspendedId contained in resumeURI contained in Email received from `Email Suspend Node` in AM
+    ///   - completion: NodeCompletion callback which returns the result of Node submission
+    func next<T>(suspendedId: String, completion:@escaping NodeCompletion<T>) {
```

Comment:
Should there be a space between the `completion:` and `@escaping` ... I'm guessing not as I see no space multiple times throughout the codebase, but I thought I'd mention it.

Reply:
That's right. Although we do not currently use any linting tool, that's one of my objects in the future to get rid of this kind of issues. Thanks for pointing that out!